package.json

```diff
@@ -36,7 +36,10 @@
   },
   "dependencies": {
     "base64-js": "^1.1.2",
+    "bluebird": "^3.5.0",
```

Comment:
Added bluebird as an explicit dependency so we can use it for a Promise polyfill. This exact version is already a transitive dependency from `request-promise`, so it's not actually adding anything new to the package.

Reply:
Hmm weird this still failed in Node 0.10 even though I'm explicitly importing Bluebird as Promise... I'll have to dig into it a bit more when I'm back in front of my computer

Reply:
Yeah let's drop this along with 0.10 and just assume native Promise support